src/Sarif.Viewer.VisualStudio.Core/OutputWindowTracerListener.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace Microsoft.Sarif.Viewer
+{
+    public class OutputWindowTracerListener : TraceListener
```

Comment:
>public class OutputWindowTracerListener : TraceListener [](http://example.com/codeflow?start=4&length=55)

Try making some updates to this file and answer the current questions. Keep in mind, this week you've got reviewers who are not as familiar with the extension and Chirs is; so we may need at a minimum some comments added to the PR to explain some of what you are doing.

If your next round of revisions don't clear things up enough; let's get on a call to go through the code.

Reply:
Please see my latest comment about this. If any question yes I can setup a call.